### ISSUE TITLE
training error  you can use the loss function `sparse_categorical_crossentropy` instead

### ISSUE THREAD

**Issue by ch4230052** [OlafenwaMoses/ImageAI#242]
Alternatively, you can use the loss function `sparse_categorical_crossentropy` instead, which does e

**Comment by ch4230052** [comment]
find the 240 line of the  __init__.py 
 change it 
 from
 model.compile(loss="categorical_crossentropy", optimizer=optimizer, metrics=["accuracy"])
to
 model.compile(loss="sparse_categorical_crossentropy", optimizer=optimizer, metrics=["accuracy"])

it will be ok
